src/common/logging.ts

```diff
@@ -1,22 +1,24 @@
 import {createLogger, format, transports} from 'winston';
-import {getEnv} from './config';
-const Sentry = require('winston-raven-sentry');
+const {Loggly} = require('winston-loggly-bulk');
+import {getEnvironmentVariables} from './config';
+import winston = require('winston');
 
 /**
  * Winston Log Object
  * Send logs to Console if message level <= 'error'.
  * Send logs to Sentry if message level is at least 'info'.
  */
+const envVariables = getEnvironmentVariables();
 const logging =
   createLogger({
-    level: getEnv() === 'production' ? 'info' : 'debug',
+    level: envVariables.env === 'production' ? 'error' : 'debug',
```

Comment:
Let's have `production` set this to `level`. Down the road, we may change it to `warn`.